packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts

```diff
@@ -297,6 +299,16 @@ export const ScreenControls = styled.div`
   }
 `
 
+export const StyledJoystreamLogo = styled(SvgJoystreamLogoFull)`
+  padding: 0.5em;
+  max-height: 2.5em;
+  height: 100%;
+
+  path {
+    fill: ${cVar('colorTextStrong')};
```

Comment:
in figma it's just `#ffffff`, also it should have box shadow

![Zrzut ekranu 2022-07-5 o 13 16 42](https://user-images.githubusercontent.com/13706246/177315824-7672c913-2f5d-478b-9935-9eec67a210e3.png)

Reply:
talked to Adam, and it should be colorTextStrong. 
Added drop shadow as well.